src/components/auth/Register.js

```diff
@@ -0,0 +1,169 @@
+import React from "react";
```

Comment:
This file is all green because I used the prettier format.

Reply:
NICE